.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Créer un rapport de bogue
+title: ''
+labels: Triage
+assignees: ''
+
+---
+
+**Décrire le bogue**
+Une description claire et concise de ce qu'est le bogue.
+
+**Reproduire**
+Etapes pour reproduire le comportement :
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** ‏Let's put accent on uppercase letters
```suggestion
Étapes pour reproduire le comportement :
```